### PR TITLE
ci: Add mcopy to action runner

### PR DESCRIPTION
**Pull request by mudler**
Follow-up of #1148

### CODE REVIEW

**Comment by Itxaka** [comment]
Is this the reason luet is failing on ci?

**Comment by mudler** [comment]
yeap, ISO are failing now https://github.com/rancher-sandbox/cOS-toolkit/runs/5640547358?check_suite_focus=true

File: .github/build.yaml.gomplate [hunk 1]
```diff
@@ -77,7 +77,7 @@
   {{{- if ne $config.luet_override "" }}}
       - name: Override luet version at toolchain/luet@{{{ $config.luet_override }}} 🔧
         run: |
-          sudo luet replace --force --yes toolchain/luet --for toolchain/luet@{{{ $config.luet_override }}}
+          sudo luet install --force --yes toolchain/luet@{{{ $config.luet_override }}}
```
Comment (mudler) [inline .github/build.yaml.gomplate:80]: this was a leftover :sweat_smile:

**Review by Itxaka — APPROVED** [review]

**Comment by mudler** [comment]
ok ISO seems to build successfully, just some tests failed (unrelated) for download issue.. but a group of them passed, so merging it

**Pull request merged**